packages/schematics/angular/pipe/index.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { addDeclarationToModule, addExportToModule } from '../utility/ast-utils';
+import {InsertChange} from '../utility/change';
+
+import {
+  Rule,
+  Tree,
+  apply,
+  branchAndMerge,
+  chain,
+  filter,
+  mergeWith,
+  move,
+  noop,
+  normalizePath,
+  template,
+  url,
+} from '@angular-devkit/schematics';
+import * as stringUtils from '../strings';
+
+import * as ts from 'typescript';
+import 'rxjs/add/operator/merge';
+
+
+function addDeclarationToNgModule(options: any): Rule {
+  return (host: Tree) => {
+    if (options['skip-import']) {
```

Comment:
This should probably be consistent with https://github.com/angular/devkit/blob/master/packages/schematics/angular/component/index.ts#L33? I'm not sure which of the two should be preferred.